test/test_utils/ec2.py

```diff
@@ -466,17 +471,19 @@ def get_ec2_instance_tags(instance_id, region=DEFAULT_REGION, ec2_client=None):
     return {tag["Key"]: tag["Value"] for tag in response.get("Tags")}
 
 
-def enforce_IMDSv2(instance_id, region=DEFAULT_REGION, ec2_client=None, hop_limit = 1):
+# If IMDSv2 is enforced on EC2 instance with hop limit 1 then IMDSv2 api calls doesn't work
+# If IMDSv2 is enforced on EC2 instance with hop limit > 1 then IMDSv2 api calls work
+@retry(stop=stop_after_attempt(16), wait=wait_fixed(60))
+def enforce_IMDSv2(instance_id, hop_limit, region=DEFAULT_REGION, ec2_client=None):
     """
-    Enabled HTTP TOKENS required option on EC2 instance with given hop limit.
+    Enable IMDSv2 on EC2 instance with hop limit > 1.
```

Comment:
will change this to earlier text `Enabled HTTP TOKENS required option on EC2 instance with given hop limit.`

Reply:
is this still applicable?